opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -253,7 +253,6 @@ class Span(trace_api.Span):
     """
 
     # Initialize these lazily assuming most spans won't have them.
-    _empty_attributes = BoundedDict(MAX_NUM_ATTRIBUTES)
     _empty_events = BoundedList(MAX_NUM_EVENTS)
     _empty_links = BoundedList(MAX_NUM_LINKS)
```

Comment:
Do `_empty_events` and `_empty_links` have the same problem?

Reply:
just fixed :). But yes, they potentially could've.